hi-ml-cpath/Makefile

```diff
@@ -176,24 +172,29 @@ define CRCKSIMCLR_SMOKE_TEST_ARGS
  --is_debug_model=True --num_workers=0
 endef
 
+define CRCK_TUNING_ARGS
```

Comment:
What about making this variable name more specific? is_finetuning was until recently only switiching ON the tuning of the encoder while here the encoder tuning is OFF

Reply:
The encoder is OFF because we use caching in TCGA-CRCK

Reply:
Note that the idea of this test is to make sure all tuning parameters work correctly, hence switching everything on (beside encoder because of caching). I wanted to make sure that given a ssl_checkpoint + src_checkpoint we are able to wire everything together properly and also ensure that deepmil checkpoint loading are not broken.